core/conversion/converters/impl/pooling.cpp

```diff
@@ -9,100 +10,163 @@ namespace converters {
 namespace impl {
 namespace {
 
-bool MaxPoolingConverter(ConversionCtx* ctx, const torch::jit::Node* n, args& args) {
+bool GlobalPoolingConverter(ConversionCtx* ctx, const torch::jit::Node* n, args& args, nvinfer1::PoolingType pool_type)
```

Comment:
Is GlobalPooling for when window size is the size of the input Tensor or when the output is a 0d tensor or something ?

Reply:
Currenly, i only route the corner case when out shape is all 1's here. There may be more, yes.